pkg/config/stepmeta.go

```diff
@@ -58,6 +58,7 @@ type StepParameters struct {
 	Conditions      []Condition           `json:"conditions,omitempty"`
 	Secret          bool                  `json:"secret,omitempty"`
 	MandatoryIf     []ParameterDependence `json:"mandatoryIf,omitempty"`
+	Hidden          bool                  `json:"hidden,omitempty"`
```

Comment:
@OliverNocon We want to add the `buildTool` to the telemetry, but our steps doesn't have it in the config. So we introduced this `hidden`, so you can add parameters without exposing them to the documentation.

WDYT? Does it make sense?

Reply:
I think this may rather complicates things. Especially telemetry does not justify this change for me.
The build tool should be already collected and one could correlate things.

We had a similar topic which is solved like that:
https://github.com/SAP/jenkins-library/blob/master/cmd/npmExecuteScripts.go#L52

Reply:
> The build tool should be already collected and one could correlate things.

That is a good point. If SWA allows this, that would be even better. @OliverNocon Could you just add the `buildTool` to the `cnbBuild` step report? Or did I understand it wrong?

> We had a similar topic which is solved like that: https://github.com/SAP/jenkins-library/blob/master/cmd/npmExecuteScripts.go#L52

Yes, this `dockerImage` part is quite straight forward. We would only need the `Hidden` for the `buildTool`.